export.py

```diff
@@ -4,28 +4,43 @@
 import csv
 import requests
 import logging
+import re
+import os
+
+def parse_csv(parsefile):
+    with open(parsefile, 'rb') as csvfile:
+        csvreader = csv.DictReader(csvfile, delimiter=',')
+        for row in csvreader:
+            request_string = 'throughput,testcase={0},traffic_type={1},' \
+                .format(row['type'], row['traffic_type'])
+            request_string += 'packetsize={0},vswitch={1},hw={2} value={3}' \
```

Comment:
as discussed, I think we don't need the HW description here, as it will be always the same